aws-integrations/firehose/README.md

```diff
@@ -12,11 +12,10 @@ For a more detailed description of the settigns and architecture of this AWS Kin
 
 | Parameter | Description | Default Value | Required |
 |---|---|---|---|
-| CoralogixRegion | The region of your Coralogix Account | _Allowed Values:_<br>- ireland<br>- stockholm<br>- india<br>- singapore<br>- us<br>- us2<br>_Default: ireland_ | :heavy_check_mark: |
+| CoralogixDomain | The Coralogix Url domain. If set, will be the url used to send telemetry (e.g. coralogix.com, eu2.coralogix.com) | | |
```

Comment:
The description of the parameter is perfectly correct, but the way it is used, I think the URL value should be 
`Url: !Sub "https://firehose-ingress.${CustomDomain}/firehose"` in the template.yaml